### ISSUE TITLE
"Entity" as Kwargs in Scheduler breaks it 

### ISSUE THREAD

**Issue by Odianosen25** [home-assistant/appdaemon#337]
Hello,

Not really an issue that needs fixing, but I have noticed that if I was to have a scheduler like this `self.run_in(self.run_scheduler, delay, entity = 'Something')`, I get the error of the scheduler being cancelled before being run. and it points to line 638 on `appdaemon.py` file (sorry don't have the error now). 

After studying the line, I noticed that the word `entity` is used there on line 634. So I simply changed to `entity_id` and it all works fine.

As I said its not really an issue, but just wanted to point this out so maybe others can avoid it.

Regards

**Comment by acockburn** [comment]
Yes, it's a known problem. At some point I intend to rename those "hidden variables" to something less likely to cause issues, so I'll leave this open to remind me.

**Comment by ReneTode** [comment]
the normal callback is 

def NTP_switch_on(self,kwargs):

so this is user error not an AD problem.
